.github/workflows/db_rollback.yml

```diff
@@ -0,0 +1,68 @@
+name: Rollback DB
+
+on:
+  workflow_dispatch:
+    inputs:
```

Comment:
Add an input for which deployed environment we're picking.  Super cool if it could be a dropdown.